contributions/catalog/webapps/templates/contribute/contribute.html

```diff
@@ -1613,6 +1623,32 @@ <h3 id="contact">Contact Details</h3>
 
 <script src="https://cdn.jsdelivr.net/npm/@tarekraafat/autocomplete.js@9.0.0/dist/js/autoComplete.min.js"></script>
 <script>
+    // toggle for first required env variable in capability
+    $("#togglePassword").on('click', function() {
```

Comment:
@sandeep-ps This function is called when in contributions edit page. The below function (which calls with bi-class) gets called on add contribution page. I'm unsure why this happens.